### DISCUSSION TITLE
Discutir trilhas do evento

### DISCUSSION THREAD

**Discussion by julianyraiol** [pythonbrasil/pybr2022-org#42] [Ideas]
- Quantas trilhas teremos no evento?
- Terá trilha em espanhol? 
- Sugestão para título das trilhas: nome de tribos indígenas de manaus

**Comment by lskbr** [comment]
+1 para nome de tribos indíginas
Fica a sugestão para nomes de peixes regionais também.
- Como mapear a dificuldade das trilhas aos nomes?

Áreas de interesse:
- Python em geral
- Criação de comunidades, experiências
- Python iniciantes
- Python na educação
- Python jogos, gráficos
- Python Ciência de Dados, Big Data
- Python Web, Cloud, Sistemas, APIs, GraphQL
- Python profissão de programador, experiência profissional
- Python Avançado: programação assíncrona, threads, metaclasses, etc
- Banco de Dados relacionais, nosql

**Comment by rennerocha** [comment]
> Terá trilha em espanhol?
Seria lindo ter uma trilha em espanhol!

**Comment by ctulio** [comment]
+2 para nome de tribos indígenas.
Fica a sugestão: 
- nome de árvores
- nome de aves
- nome de rios
- misturar elementos da cultura local (animais, comidas, lendas, rios)

> Como mapear a dificuldade das trilhas aos nomes?

[O jiu-jitsu brasileiro tem raízes em Manaus](http://ge.globo.com/am/noticia/2014/10/de-heranca-niponica-paixao-local-o-elo-centenario-entre-jiu-jitsu-e-manaus.htm).
Poderia ser usado a escala de graduação do jiu-jitsu (ou uma versão simplificada) para identificar as palestras, não as trilhas.
![image](https://i.pinimg.com/736x/4b/b4/32/4bb43244e573d04c60f91e8840b21aa2.jpg)

**Comment by lskbr** [comment]
eu gostaria muito de ter algo da mitologia amazonense.

Envoyé de mon iPhone

**Comment by lskbr** [comment]
Quanto a trilha em espanhol, precisamos de um voluntário fluente na língua para fazer anúncios nas comunidades da America Latina e Espanha.
Uma trilha em Inglês também pode ser aberta.
